engine/orchestration/test/com/cloud/vm/VirtualMachineManagerImplTest.java

```diff
@@ -505,4 +513,24 @@ public void testSendStopWithNullAnswer() throws Exception {
 
         Assert.assertFalse(actual);
     }
+
```

Comment:
@swill I'll fix the merge conflicts and test that build succeeds for both these PRs